Components/Atoms/GoalListItem.js

```diff
@@ -65,7 +70,78 @@ const GoalButtonFront = ({ text, totalCount, pushGoal, lastTimeStampMessage, is_
     </TouchableWithoutFeedback>
   );
 };
+const GoalButtonFront = ({ text, totalCount, pushGoal, lastTimeStampMessage, is_overdue }) => {
+  const color = "#17355A"; //"#2DAAFF"; //is_overdue ? "red" : "green";
+  const color_shade = is_overdue ? "#C94818" : "#193162";
+  const main_background = "#FFF9FD";
+  return (
+    <TouchableWithoutFeedback onPress={pushGoal}>
+      <AwesomeButtonProgress
+        progress
+        height={115}
+        activityColor={main_background}
+        backgroundColor={main_background}
+        backgroundShadow={main_background}
+        backgroundDarker={main_background}
+        backgroundPlaceholder={main_background}
+        backgroundActive={main_background} //"#C0C0C0"}
+        backgroundProgress={main_background}
+        activeOpacity={70}
+        borderColor={main_background}
+        progressLoadingTime={10000}
+        width={Dimensions.get("window").width * 85}
+        raiseLevel={0}
+        action={(element, next) => doSomethingThenCall(next)}
+        onPress={next => {
+          pushGoal();
+          next();
+        }}
+      >
+        <Neomorph
+          darkShadowColor={color_shade} //"#D1CDC7" // <- set this
+          lightShadowColor="#FFF" //{color_shade} ///
+          style={styles.neomorph}
+        >
+          <View style={styles.topRow}>
+            <Text style={styles.task}>{text}</Text>
+            <View style={[styles.dash, { borderColor: color }]}>
+              <Text style={[styles.frequency, { textDecorationColor: color, color: color }]}>
```

Comment:
Are we not updating the text-decoration color based on whether it's overdue still?